docs/blog/2019-06-21-dropping-support-for-node-6/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Dropping Support for Node 6"
+date: 2019-06-21
+author: Dustin Schau
+excerpt: "We're dropping support for Node 6 in minor releases of core Gatsby packages. Learn more about why this decision was made."
+tags:
+  - core
+  - deprecation
+---
+
+Effective in Gatsby v2.10.0, we are making a potentially breaking change and dropping support for Node 6. We don't take this decision lightly, and realize that there is _potential_ for breaking applications by not following [semantic versioning][semver] _strictly_ and releasing Gatsby v3.0.0. However, we feel that this decision is in the best interest of our users and enables the Gatsby Core team to focus more efficiently on delivering fixes and features. In this post, I outline the reasoning for why we're making this decision and how it potentially impacts you, a Gatsby user.
```

Comment:
```suggestion
Effective in Gatsby v2.10.0, we are making a potentially breaking change and dropping support for Node 6. We don't take this decision lightly, and realize that there is _potential_ for breaking applications by not releasing this as releasing Gatsby v3.0.0 (if following [semantic versioning][semver] _strictly_). However, we feel that this decision is in the best interest of our users and enables the Gatsby Core team to focus more efficiently on delivering fixes and features. In this post, I outline the reasoning for why we're making this decision and how it potentially impacts you, a Gatsby user.
```

Reply:
Not sure this is meaningfully improved. Could you provide a little more context?

> We don't take this decision lightly, and realize that there is _potential_ for breaking applications by not releasing this as releasing Gatsby v3.0.0 (if following [semantic versioning][semver] _strictly_)

> We don't take this decision lightly, and realize that there is _potential_ for breaking applications by not following [semantic versioning][semver] _strictly_ and releasing Gatsby v3.0.0